physics/GFS_PBL_generic.F90

```diff
@@ -134,6 +137,7 @@ subroutine GFS_PBL_generic_pre_run (im, levs, nvdiff, ntrac,
               vdftra(i,k,4) = qgrs(i,k,ntoz)
             enddo
           enddo
+          rtg_ozone_index = 4
```

Comment:
I am not happy with hardcoded indices set in the scheme, but it's already in the existing code. Is there anything we can do better (in the future)? Note also that we could remove all `wsm6` code, since we do not support this option.

Reply:
The indices are hard-coded in that routine, and rtg_ozone_index's purpose is to pass the index to other routines that need it. Until the hard-coded indices are removed from the file, there is no way to replace rtg_ozone_index.